doc/adr/0002-GraphQL.md

```diff
@@ -14,6 +14,36 @@ Date: 2022-03-15
 * We then [opened this issue][issue] in early January 2021.
 * [Alternatives to going from current REST to GraphQL][alt] seem limited; [more][more].
 
+#### Use cases yet to support
+
+These are [high level][org] (non-TSP specific), so will require focused [TSP][tsp] prototyping.
+
+1. ***Describe*** client schemas based on potentially refactored [domain][language] resolutions.
+1. ***Dimension*** client requests towards trace servers solely out of the needed information.
+1. Get potentially more ***predictable*** and scaled results with only the requested data ranges.
+1. ***Combine*** more than just one trace data source or server in request if resolvable so.
+1. Enable both API compatibility and ***evolution***, through being version-agnostic in requests.
+1. Allow potential ***custom*** resolution logic between clients and trace data sources, at scale.
+1. [Keep REST support or ***iterate*** towards GraphQL][iter] use in virtually any TSP client.
+
+#### And their stakeholders
+
+There is no mapping between these roles and aforementioned use cases, as many may apply to many.
+
+1. TSP ***client*** developers, users or integrators; [Python][client-py], [TypeScript][client-ts].
+1. Tracing ***application*** developers, either [Theia][theia] or Eclipse (or else, potentially).
+1. Tracing ***solution*** architects and marketeers, potentially.
+1. Tracing ***users*** in general;
+   * either indirectly through solution usage,
+   * or directly if coding queries against tracing servers.
```

Comment:
That's only time when a user has direct impact because the queries are different. Otherwise users won't be aware if it's GraphQL, REST or potential other.

Reply:
Unless they're aware of the said solution (its architecture). I'm amending these few lines now; thanks.